middleware/swagger-validator.js

```diff
@@ -361,12 +368,14 @@ exports = module.exports = function (options) {
                     var schema = swaggerVersion === '1.2' ? parameter : parameter.schema;
                     var pLocation = swaggerVersion === '1.2' ? schema.paramType : schema.in;
                     var val;
+                    var originalVal;
 
                     paramName = schema.name;
                     paramPath = swaggerVersion === '1.2' ?
                       req.swagger.operationPath.concat(['params', paramIndex.toString()]) :
                       parameter.path;
                     val = req.swagger.params[paramName].value;
+                    originalVal = req.swagger.params[paramName].originalValue;
```

Comment:
**question:**  This looks like the change would become a minor/feature now since the API has changed adding more info to the generated params object?

Reply:
Nah, the generated params object already had `originalValue`.

Reply:
Ah i misread the code as the other way around